posthog/temporal/workflows/bigquery_batch_export.py

```diff
@@ -161,7 +165,7 @@ async def insert_into_bigquery_activity(inputs: BigQueryInsertInputs):
                     jsonl_file.write_records_to_jsonl([row])
 
                     if jsonl_file.tell() > settings.BATCH_EXPORT_BIGQUERY_UPLOAD_CHUNK_SIZE_BYTES:
-                        activity.logger.info(
+                        logger.info(
```

Comment:
debug?

Reply:
I'll answer here once, but this applies to all your comments: I dislike your one word review comments.

It's very hard to answer if you don't provide any points for me to answer. Are you expecting me to answer with a single word (like "no") too? I assume you expect some explanation, so please include some yourself. Otherwise I need to spend time trying to figure out what went through your mind first, before I even dive into an answer. If a comment is not worth describing, then reconsider if it should be a comment in the first place.

I can help with a question: Why do you think this should be debug level?

Reply:
I explained on an earlier comments - that it could be noisy. To elaborate on this: if we use only a single log level, then we can't easily separate logs, what's the point of having multiple log levels if we never use them? Specifically this is much less of a problem for us, but more of a problem for the user. Note that if we want debug log lines that aren't meant for the user, then they shouldn't be visible for the user and we can use Loki. As a user if you think about the UI, then I only have access to checkboxes for what log levels I want to see. It would be nice for the user if they can have some more detailed level logs filtered out - the easiest way seems to be to make them debug level and then the checkbox can be clicked in the UI. In the app logs everything was/too many things maybe still are info level, so it's hard for the user to reduce the noise & get a higher level overview.